packages/now-cli/src/commands/env/ls.ts

```diff
@@ -102,7 +91,9 @@ function getRow(env: ProjectEnvVariableV5) {
   return [
     chalk.bold(env.key),
     value,
-    env.target || '',
+    (Array.isArray(env.target) ? env.target : [env.target || ''])
+      .map(capitalize)
```

Comment:
Its a little confusing that these are printed as titlecase in `vc env ls` but the `vc env add` only accepts lowercase.

Perhaps we should change the users input to lowercase or just remove this titlecase to avoid confusion?

Reply:
I followed the spec but I agree this is a compelling point to remove the titlecase in the list 👍

Reply:
I was thinking of someone who did `vc env ls | grep 'production' | cut -d ' ' | vc env rm` or something where they read `ls` output to send as input into `rm`.

Maybe its okay since the error message seems to use lowercase.